frontend/package.json

```diff
@@ -54,6 +54,7 @@
     "swr": "^2.2.4",
     "tailwind-merge": "^2.2.2",
     "tailwindcss-animate": "^1.0.7",
+    "vaul": "^0.9.1",
```

Comment:
@im-adithya how did you install it? did you do it through shadcn? there should be an update to the components json

Reply:
We use this in the [Drawer](https://ui.shadcn.com/docs/components/drawer) (See Manual installation option, I followed just that)

Reply:
hmm, I think the CLI makes some other changes, I think we should consistently use it? CC @reneaaron

Reply:
Just tried using `npx shadcn-ui@latest add drawer`, it does nothing more but nicely does what's to be done manually